opendbc/car/body/carstate.py

```diff
@@ -30,10 +31,12 @@ def update(self, cp, *_) -> structs.CarState:
     return ret
 
   @staticmethod
-  def get_can_parser(CP):
+  def get_can_parsers(CP):
     messages = [
       ("MOTORS_DATA", 100),
       ("VAR_VALUES", 10),
       ("BODY_DATA", 1),
     ]
-    return CANParser(DBC[CP.carFingerprint]["pt"], messages, 0)
+    return {
+      Bus.MAIN: CANParser(DBC[CP.carFingerprint][Bus.MAIN], messages, 0)
+    }
```

Comment:
```suggestion
    return {Bus.MAIN: CANParser(DBC[CP.carFingerprint][Bus.MAIN], messages, 0)}
```

this is fine, and GH shows the diff better